src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -56,59 +56,47 @@ module.exports = class ArtworkCommercialView extends Backbone.View
 
     loggedInUser = CurrentUser.orNull()
 
-    # Show the new buy now flow if you have the lab feature or feature flag enabled
-    if sd.ENABLE_NEW_BUY_NOW_FLOW || loggedInUser?.hasLabFeature('New Buy Now Flow')
-      analytics.track('Click', {
-        subject: 'buy',
-        type: 'button',
-        flow: 'buy now'
-      })
-
-      serializer = new Serializer @$('form')
-      data = serializer.data()
-      editionSetId = data.edition_set_id
-      $target = $(e.currentTarget)
-
-      # If this artwork has an edition set of 1, send that in the mutation as well
-      if @artwork.get('edition_sets')?.length && @artwork.get('edition_sets').length == 1
-        editionSetId = @artwork.get('edition_sets')[0] && @artwork.get('edition_sets')[0].id
-
-      if loggedInUser
-        $target.attr 'data-state', 'loading'
-        createOrder
-          artworkId: @artwork.get('_id')
-          editionSetId: editionSetId
-          quantity: 1
-          user: loggedInUser
-        .then (data) ->
-          { order, error } = data?.ecommerceCreateOrderWithArtwork?.orderOrError || {}
-          if order
-            location.assign("/orders/#{order.id}/shipping")
-          else
-            console.error('createOrder', error)
-            $target.attr 'data-state', 'loaded'
-            errorModal.renderBuyNowError(error)
-        .catch (err) ->
-          console.error('createOrder', err)
+    analytics.track('Click', {
+      subject: 'buy',
+      type: 'button',
+      flow: 'buy now'
+    })
+
+    serializer = new Serializer @$('form')
+    data = serializer.data()
+    editionSetId = data.edition_set_id
+    $target = $(e.currentTarget)
+
+    # If this artwork has an edition set of 1, send that in the mutation as well
+    if @artwork.get('edition_sets')?.length && @artwork.get('edition_sets').length == 1
+      editionSetId = @artwork.get('edition_sets')[0] && @artwork.get('edition_sets')[0].id
+
+    if not loggedInUser
```

Comment:
`if not loggedinUser` sorta seems weird to me? maybe better to just do `if loggedinUser .... else` (switch the order).